src/txn/txn_recover.c

```diff
@@ -84,6 +84,12 @@ __recovery_cursor(
             WT_RET(__wt_open_cursor(session, r->files[id].uri, NULL, cfg, &c));
             r->files[id].c = c;
         }
+#ifndef WT_STANDALONE_BUILD
+        /* In the even of a clean shutdown, there shouldn't be any other table log records other
```

Comment:
typo: even -> event
Also multiline comments should have blank lines with just /* and */ on them.

Reply:
Fixed.